src/roles/common/tasks/set-major-minor-versions.yml

```diff
@@ -56,7 +56,7 @@
       - from_major_version == to_major_version
       - from_minor_version == to_minor_version
       - (from_patch_version == to_patch_version) or (from_major_version|int >= 6)
-      - (upgrade_from_version|upper|replace('R', '')).split('U')[0] is version_compare('5.4.1', operator='ge', strict=True)
```

Comment:
I think version_compare was renamed to version in Ansible 2.5, we should probably use that here and in other parts of the file

Reply:
version_compare is still the correct filter. Since ansible 2.5 what they changed is you cannot use `| version_compare`, you need to use `is version_compare`

Reply:
Okay, I saw the documentation now. Will fix it.

Reply:
There's a section here for comparing versions: https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html

version_compare still works, but the docs point to using version instead. Maybe we don't need to change version_compare to version but I think there are some places where we use version - we should be consistent either way, right?

Reply:
I have changed it. Good to be consistent and future proof.